src/index.js

```diff
@@ -48,7 +52,9 @@ export default function loader(src) {
 
     const encodedData = options.generator
       ? options.generator(src)
-      : `data:${mimetype || ''};${encoding},${src.toString(encoding)}`;
+      : `data:${mimetype || ''}${encoding ? `;${encoding}` : ''},${
+          encoding ? src.toString(encoding) : src.toString()
```

Comment:
Can be simplified to `src.toString(encoding)`, when `encoding` is undefined it will be as `src.toString()`

Reply:
`encoding` here is never undefined it's 'base64' (if it was already undefined or true)[https://github.com/webpack-contrib/url-loader/pull/210/files#diff-1fdf421c05c1140f6d71444ea2b27638R40] or a string value or `false` and `false` will cause a bug.